.goreleaser.yml

```diff
@@ -56,3 +56,18 @@ dockers:
     extra_files:
       - etc/chart_schema.yaml
       - etc/lintconf.yaml
+brews:
+  - github:
```

Comment:
quick note while I was looking… when we upgrade goreleaser to `v0.139.0` we'll want to change this to `tap` https://goreleaser.com/deprecations/#brewsgithub but 👌 for now!